### ISSUE TITLE
Feature request: Keyboard echo profile

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#15521]
### Is your feature request related to a problem? Please describe.

I had a request from someone who only wanted to hear keys echoed.  While that (without other screen reading) is likely an edge case, it got me thinking about a keyboard echo profile.  I can see the benefit of having the ability to create a different profile for keyboard echo as some users may prefer a different voice, or faster or slower speech rate for keyboard echo.

### Describe the solution you'd like

The ability to create a configuration profile for keyboard echo.

### Describe alternatives you've considered

This could potentially also include word echo, I don't have a strong opinion either way.

**Comment by seanbudd** [comment]
NVDA cannot load multiple synthesizers or profiles at the same time. Loading profiles and switching synthesizers for keyboard  echo would be extremely slow and annoying with our current design

**Comment by LowieB2727** [comment]
I emailed NVDA but I am desperate for this type of functionality - only hearing the keys I press in real time, but NOTHING else. How do I configure NVDA to do this? I used MS Narrator for over a year as a data entry clerk and it reduced my error rate by about 25%, and increased my productivity by about the same. But that was on an older version on Windows XP in 2015. It used to allow me to disable all other audio notifications (what Excel cell I selected, telling me I am in "Edit" mode in the cell, etc...), but not anymore. The importance of this feature is that I would be reading/seeing the character (number) I was typing and hearing what I typed as well. When I made a mistake, they wouldn't match and I would know immediately I made a mistake without having to look back at what I actually typed and proofread. So, I only want to hear each keyboard character spoken as it is pressed. I can't find ANY software on the planet that does only this very simple task and Nothing else. Any ideas???? Thanks!
